deps.edn

```diff
@@ -6,7 +6,8 @@
  ;; https://github.com/pedestal/pedestal#getting-the-latest-release
  :deps {io.pedestal/pedestal.service {:mvn/version "0.5.10"}
         io.pedestal/pedestal.route {:mvn/version "0.5.10"}
-        io.pedestal/pedestal.jetty {:mvn/version "0.5.10"}}
+        io.pedestal/pedestal.jetty {:mvn/version "0.5.10"}
+        org.clojure/tools.analyzer {:mvn/version "1.1.0"}}
```

Comment:
Added to avoid warnings in with Clojure 1.11

```
clj -A:dev
WARNING: update-vals already refers to: #'clojure.core/update-vals in namespace: clojure.tools.analyzer.utils, being replaced by: #'clojure.tools.analyzer.utils/update-vals
...
```